helm/cluster-api-provider-azure/files/infrastructure/patches/versions/v1beta1/azuremachinepools.infrastructure.cluster.x-k8s.io.yaml

```diff
@@ -137,6 +137,52 @@
                 image:
                   description: Image is used to provide details of an image to use during VM creation. If image details are omitted the image will default the Azure Marketplace "capi" offer, which is based on Ubuntu.
                   properties:
+                    computeGallery:
```

Comment:
This replaces now deprecated SharedGallery. Giant Swarm vintage workload clusters are not using these fields, so they are not affected with this change.